docs/articles/getting-started/simple-example.md

```diff
@@ -161,11 +161,11 @@ flow.map(initial_count)
 flow.reduce_epoch(add)
 ```
 
-Its super power is that it can repeatedly combine together items into a single, aggregate value via a reducing function. Think about it like reducing a sauce while cooking; you are boiling all of the values down to something more concentrated.
+Its super power is that it can repeatedly combine together items into a single, aggregate value via a reducing function. Think about it like reducing a sauce while cooking: you are boiling all of the values down to something more concentrated.
 
 In this case, we pass it the reducing function `add()` which will sum together the counts of words so that the final aggregator value is the total.
 
-How does reduce epoch know which items to combine? Part of its requirements are that the input items from the previous step in the dataflow are `(key, value)` two-tuples, and it will make sure that all values for a given key are passed to the reducing function, but two separate keys will never have their values mixed. Thus, if we make the word the key, we'll be able to get separate counts!
+How does reduce epoch know which items to combine? Part of its requirements are that the input items from the previous step in the dataflow are `(key, value)` two-tuples, and it will make sure that all values for a given key are passed to the reducing function. Thus, if we make the word the key, we'll be able to get separate counts!
```

Comment:
This felt unnecessary to me since the conversation around `keys` already seems to imply the grouping.